gui/wxpython/datacatalog/tree.py

```diff
@@ -981,17 +982,31 @@ def OnDoubleClick(self, node):
                     dlg = wx.MessageDialog(
                         parent=self,
                         message=_(
-                            "Map <{0}@{1}> is not in the current location"
-                            " and therefore cannot be displayed."
-                            "\n\n"
-                            "To display this map switch to mapset <{1}> first."
+                            "Map <{0}@{1}> is not in the current location. "
+                            "Do you want to switch to the location '{2}'? "
+                            "Your currently displayed layers will disappear."
```

Comment:
The reason I used the word "disappear" in the original issue is that it expresses well the confusion a user may have when the layers are unloaded, i.e., something unexpected happened. Furthermore, "disappear" in this context may sound as "underlying data will be deleted". I don't have another suggestion at this point, but Perhaps mentioning again the reason for it in connection with the change in wording can bring some understanding.

Reply:
Maybe something along the lines "All current Map Displays will be closed"?

Reply:
And put it on the next line? Now it looks as follows: ![switch](https://user-images.githubusercontent.com/49241681/120106511-78ef4c80-c15d-11eb-92da-0623421ba425.PNG)